README.md

```diff
@@ -452,14 +452,15 @@ export class User extends Entity {
 
 ### Default Clause/Constant
 
-Use the `default` property to have MySQL handle setting column `DEFAULT` value.
+Use the `default` and `dataType` properties to have MySQL handle **setting column `DEFAULT` value**.
```

Comment:
I think we probably want to show this table: https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html#data-mapping-properties. Users can set the connector specific configurations/properties on the model properties.

Reply:
We have them documented in LB4 files: https://loopback.io/doc/en/lb4/Model.html#how-loopback-models-map-to-database-tablescollections. I will add that link, then introduce the rest MySQL-specific settings